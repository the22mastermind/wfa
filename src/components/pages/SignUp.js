import React from 'react';

const SignUp = () => {
  return (
    <div className="container">
      <form className="form-wrapper">
        <div className="form-title">Sign Up</div>
        <div className="names">
          <input type="text" required placeholder="First name" className="input"/>
          <input type="text" required placeholder="Last name" className="input"/>
        </div>
        <div className="input-wrapper">
          <input type="email" required placeholder="Email" className="input"/>
        </div>
        <div className="input-wrapper">
          <input type="telephone" required placeholder="Phone" className="input"/>
        </div>
        <div className="input-wrapper">
          <input type="password" required placeholder="Password" className="input"/>
        </div>
        <div className="input-wrapper">
          <input type="password" required placeholder="Confirm password" className="input"/>
        </div>
        <div className="submit-btn-wrapper">
          <input type="submit" className="submit-btn" value="Sign up"/>
        </div>
        <p className="alt-wrapper">
          <span className="alt-text">Already registered?</span>
          <a href="/signin" className="link">Sign in</a>
        </p>
      </form>
    </div>
  );
}

export default SignUp;
