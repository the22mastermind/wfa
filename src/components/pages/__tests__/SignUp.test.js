import React from 'react';
import renderer from 'react-test-renderer';

import SignUp from '../SignUp';


it('matches the Sign Up snapshot', () => {
  const tree = renderer.create(<SignUp/>).toJSON();
  expect(tree).toMatchSnapshot();
});
