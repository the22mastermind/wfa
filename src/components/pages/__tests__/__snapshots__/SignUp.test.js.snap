// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the Sign Up snapshot 1`] = `
<div
  className="container"
>
  <form
    className="form-wrapper"
  >
    <div
      className="form-title"
    >
      Sign Up
    </div>
    <div
      className="names"
    >
      <input
        className="input"
        placeholder="First name"
        required={true}
        type="text"
      />
      <input
        className="input"
        placeholder="Last name"
        required={true}
        type="text"
      />
    </div>
    <div
      className="input-wrapper"
    >
      <input
        className="input"
        placeholder="Email"
        required={true}
        type="email"
      />
    </div>
    <div
      className="input-wrapper"
    >
      <input
        className="input"
        placeholder="Phone"
        required={true}
        type="telephone"
      />
    </div>
    <div
      className="input-wrapper"
    >
      <input
        className="input"
        placeholder="Password"
        required={true}
        type="password"
      />
    </div>
    <div
      className="input-wrapper"
    >
      <input
        className="input"
        placeholder="Confirm password"
        required={true}
        type="password"
      />
    </div>
    <div
      className="submit-btn-wrapper"
    >
      <input
        className="submit-btn"
        type="submit"
        value="Sign up"
      />
    </div>
    <p
      className="alt-wrapper"
    >
      <span
        className="alt-text"
      >
        Already registered?
      </span>
      <a
        className="link"
        href="/signin"
      >
        Sign in
      </a>
    </p>
  </form>
</div>
`;
